### PR TITLE
update skeleton files to match exercise order

### LINKED ISSUE #318: Adjust skeleton files to match exercise order
Right now `hello-world` and `bob` start with a skeleton file, but the [order of exercises](https://github.com/exercism/xjavascript/blob/master/config.json#L8-L12) has changed to 

0. hello-world
0. leap
0. hamming
0. rna-transcription
0. bob

It would probably be a good idea to remove the skeleton file from the Bob exercise, and add it to leap.

### PR DESCRIPTION
**Pull request by joshuacolvin**
This resolves #318

### CODE REVIEW

**Comment by rchavarria** [comment]
Yep! Many, many thanks!

**Comment by joshuacolvin** [comment]
No problem! Glad to help!

File: exercises/leap/leap.js [hunk 1]
```diff
@@ -3,12 +3,12 @@
 // convenience to get you started writing code faster.
```
Comment (kytrinyx) [inline exercises/leap/leap.js]: The comment refers to the "Bob" exercise.

Comment (rchavarria) [inline exercises/leap/leap.js]: Whoops! Embarrassing, I didn't notice.

Would you mind @joshuacolvin to replace "Bob" with "Leap"? A tiny PR would be ok. Thanks.

Comment (joshuacolvin) [inline exercises/leap/leap.js]: So sorry! Can't believe I didn't notice either!

Fixed in #325

**Pull request merged**